webroot/img/flags/nap.svg

```diff
@@ -3,16 +3,13 @@
    version="1.0"
    width="600"
    height="400"
-   id="svg698"
    sodipodi:docname="Flag_of_Naples.svg"
    xml:space="preserve"
    inkscape:version="1.2 (dc2aedaf03, 2022-05-15)"
    xmlns:inkscape="http://www.inkscape.org/namespaces/inkscape"
    xmlns:sodipodi="http://sodipodi.sourceforge.net/DTD/sodipodi-0.dtd"
    xmlns="http://www.w3.org/2000/svg"
    xmlns:svg="http://www.w3.org/2000/svg"><defs
-     id="defs702" /><sodipodi:namedview
```

Comment:
You deleted a bit too much here. (Or too little, depending on how one looks at it.)
Only the `id="defs702"` part is the id attribute. The `/>` closes the preceding `<defs` and then the following `<sodipodi:namedview` starts a new element.
So after deleting this line, all the attributes that were part of the `sodipodi:namedview` element are now attached to the `defs` element.

Reply:
I got rid of the trailing <defs I left alone, maybe that'll settle it? Or should I readd everything except id="svg698"?

Reply:
> Or should I readd everything except id="svg698"?

That would be the option least likely to cause breakage. Right now, the attributes that used to belong to `<sodipodi:namedview` are just awkwardly hanging around without a tag.